app/views/games/show.html.erb

```diff
@@ -1,4 +1,52 @@
 <div class ="game-show">
+  <div class="container-fluid">
+  <div class="row">
+    <div class="col-md-6 col-sm-6">
+      <h2 style="font-weight:bold; margin-bottom: 20px; color: #EF7030;"><%= @game.title %></h2>
+    <p><%= @game.description %></p>
+    <!-- game management begins -->
+    <% if (user_is_dev? && @game.user_id == current_user.id ) ||
+      user_is_admin? %>
+      <p>Type:
+        <% @game.taggings.map do |tagging| %>
+        <%= Tag.find(tagging.tag_id).name %> &nbsp;|&nbsp;
+        <% end %>
+      </p>
+      <div class="container-fluid" style="background-color: steelblue">
+    <%# if can?(:manage, @game) %>
+    <% if user_is_admin? ||
+      ( user_is_dev? && @game.user_id == current_user.id) %>
```

Comment:
I have a ticket to refactor all of the custom permissions methods out and use CanCanCan. Don't worry about it for this ticket. There's a lot of this to change in the entire app.